services/eth_gateway.go

```diff
@@ -30,6 +30,7 @@ type Eth struct {
 	ClaimUnusedPRLs     ClaimPRLs
 	GenerateEthAddr     GenerateEthAddr
 	BuryPrl             BuryPrl
+	ClaimPrl            ClaimPRL
```

Comment:
So just so I'm understanding things, ClaimPRL is what will actually call the smart contract claim() method, and ClaimPRLs will be retrieving the unused PRLs from the transaction address?  

We might should change the name of ClaimPRLs back to ClaimUnusedPRLs or call it something else like RetrieveUnusedPRLs, just so it's really clear that it is not invoking the smart contract's claim() function, but just doing a transaction with PRL tokens.